iati/validator.py

```diff
@@ -331,16 +399,29 @@ def _check_rules(dataset, ruleset):
         dataset (iati.core.data.Dataset): The Dataset to check Ruleset conformance with.
         ruleset (iati.code.Ruleset): The Ruleset to check conformance with.
 
-
     Returns:
         iati.validator.ValidationErrorLog: A log of the errors that occurred.
 
     """
     error_log = ValidationErrorLog()
+    error_found = False
 
-    if not ruleset.is_valid_for(dataset):
-        error = ValidationError('err-ruleset-conformance-fail', locals())
+    for rule in ruleset.rules:
+        result_rule_validation = rule.is_valid_for(dataset)
```

Comment:
`result_rule_validation` seems a bit long as a name... maybe something like `validation_status` (though that isn't wonderful either)

Reply:
changed in 53b9e2b7341da8a73009d5a8848c5e94d64a8e54